starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -979,9 +979,20 @@ sub check_bright_perigee {
         }
     }
 
-    # Pass 1 to _guide_count as third arg to use the count_9th mode
-    my $bright_count = sprintf("%.1f",
-        call_python("utils._guide_count", [ \@mags, $self->{ccd_temp}, 1 ]));
+    # Pass 1 to _guide_count as fourth arg to use the count_9th mode
```

Comment:
This comment is no longer accurate but can be removed now with the kwargs explicitly called out.